src/providers/twitch/TwitchMessageBuilder.cpp

```diff
@@ -917,12 +918,13 @@ void TwitchMessageBuilder::parseHighlights(bool isPastMsg)
         if (!isPastMsg)
         {
             if (playSound &&
-                (!hasFocus || getSettings()->highlightAlwaysPlaySound))
+                (!hasFocus || getSettings()->highlightAlwaysPlaySound) &&
```

Comment:
these if-statements could use some cleanup so it's possible to read what they do